### PR TITLE
[Issue 14462] 500th contributor blog

### PR DESCRIPTION
**Pull request by alicebi-streamnative**
*(If this PR fixes a github issue, please add `Fixes #<xyz>`.)*

Fixes ISSUE-14462

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: [ISSUE-14462](https://github.com/streamnative/pulsar/issues/3824)

### Motivation

Recently, the Pulsar community reached 500 contributors. This blog, co-authored by Matteo Merli with Karin Landers and Alice Bi, provides an overview of the growth of the Pulsar community, product updates, and celebrates this community milestone.

### Modifications

Adding this blog to https://pulsar.apache.org/blog/

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

- [x] This change is a trivial rework / code cleanup without any test coverage.

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API: (no)
  - The schema: (no)
  - The default values of configurations: (no)
  - The wire protocol: (no)
  - The rest endpoints: (no)
  - The admin cli options: (no)
  - Anything that affects deployment: (no)

### Documentation

Check the box below or label this PR directly (if you have committer privilege).

Need to update docs? 

- []  `doc-required`

### CODE REVIEW

File: site2/website/blog/2022-02-24-apache-pulsar-community-welcomes-500th-contributor [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+![pulsar contributor history](https://imgur.com/a/ECFl6Pi)
+
+Since the Pulsar project gained its 400th contributor and surpassed Kafka for the total number of commits in June 2021, the Pulsar community has released major product updates and ecosystem enhancements to improve developer experience and lower the barrier to adoption. Recent product feature updates include:
+
+## Product Updates and Ecosystem Development
+
+The Apache Pulsar community released versions [2.8.0](https://pulsar.apache.org/blog/2021/09/23/Apache-Pulsar-2-8-1/) in June 2021, and [2.9.0](https://pulsar.apache.org/release-notes/#281-mdash-2021-09-10-a-id281a) in December 2021, which contain a number of new benefits, including: 
+
+1. **Enhanced support for protocol handlers** enables teams to integrate existing applications built on other technologies with Pulsar without code changes. Popular protocol handlers developed by the community including:
```
Comment (dave2wave) [inline site2/website/blog/2022-02-24-apache-pulsar-community-welcomes-500th-contributor:24]: True, but here you are highlighting SN product ...

File: site2/website/blog/2022-02-24-apache-pulsar-community-welcomes-500th-contributor [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+4. **Improved multi-tenancy support with tenant and namespace rate limiting.** [[PIP-82]](https://github.com/apache/pulsar/pull/11918) This feature allows users to configure a namespace or tenant wide limit for producers and consumers, and have that enforced irrespective of the number of topics in the namespace, allowing control of the max rate across a given namespace. 
+
+5. **Simplified microservices development** with Pulsar’s high-level APIs is now possible. [Exclusive producers](https://www.youtube.com/watch?v=Chw8VBS9WfI) enable developers to build coordination and consensus mechanisms.
+
+6. The new [Transactions feature](https://streamnative.io/blog/release/2021-06-14-exactly-once-semantics-with-transactions-in-pulsar/) empowers organizations to use Pulsar as a system-of-record that enables event-based architectures, similar to those popular in finance and real-time systems. With this update, Pulsar's message delivery/processing semantic evolved from message deduplication on a single topic, to atomic produce and acknowledgment over multiple topics.
```
Comment (dave2wave) [inline site2/website/blog/2022-02-24-apache-pulsar-community-welcomes-500th-contributor:36]: Again you are referring to a SN blog post which does not link back in any way to the Apache Pulsar project.

File: site2/website/blog/2022-02-24-apache-pulsar-community-welcomes-500th-contributor [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+7. [Function Mesh for Apache Pulsar](https://www.youtube.com/watch?v=fAn1HZdb50k) is now available. With Function Mesh, teams can better manage functions, connectors, and applications composed of multiple functions directly in Kubernetes.
+
+## What’s Next? 
+
+This year you can expect to see more exciting updates in the Pulsar project, including:
+
+1. [Moving Toward a ZookKeeper-Less Apache Pulsar](https://streamnative.io/blog/release/2022-01-25-moving-toward-a-zookeeperless-apache-pulsar/) Since its inception, Pulsar has used ZooKeeper as its distributed coordinator to store critical metadata information. Pulsar Improvement Plan [[PIP-45]](https://github.com/apache/pulsar/wiki/PIP-45%3A-Pluggable-metadata-interface) eliminates the ZooKeeper dependency and replaces it with a pluggable framework. This plug-and-play framework enables you to reduce the infrastructure footprint of Pulsar by leveraging alternative metadata and coordination systems based upon your deployment environment, resulting in improved consistency, resilience, stability, and reducing the technical debt of deployment. The end result is that starting in version 2.10, you can now run Pulsar without ZooKeeper.
```
Comment (dave2wave) [inline site2/website/blog/2022-02-24-apache-pulsar-community-welcomes-500th-contributor:44]: Again you are referring to a SN blog post which does not link back in any way to the Apache Pulsar project.

File: site2/website/blog/2022-02-24-apache-pulsar-community-welcomes-500th-contributor [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+1. [Moving Toward a ZookKeeper-Less Apache Pulsar](https://streamnative.io/blog/release/2022-01-25-moving-toward-a-zookeeperless-apache-pulsar/) Since its inception, Pulsar has used ZooKeeper as its distributed coordinator to store critical metadata information. Pulsar Improvement Plan [[PIP-45]](https://github.com/apache/pulsar/wiki/PIP-45%3A-Pluggable-metadata-interface) eliminates the ZooKeeper dependency and replaces it with a pluggable framework. This plug-and-play framework enables you to reduce the infrastructure footprint of Pulsar by leveraging alternative metadata and coordination systems based upon your deployment environment, resulting in improved consistency, resilience, stability, and reducing the technical debt of deployment. The end result is that starting in version 2.10, you can now run Pulsar without ZooKeeper. 
+
+2. **Tiered storage improvement** that includes native offloading into lakehouse formats like Delta Lake, Apache Hudi, and Apache Iceberg is also expected in future releases. These technologies allow for data access patterns in addition to streams and enables Pulsar to offer more efficient batch access.
```
Comment (dave2wave) [inline site2/website/blog/2022-02-24-apache-pulsar-community-welcomes-500th-contributor:46]: I've been investigating Tiered storage and wonder why I missed this on https://pulsar.apache.org/

**Comment by dave2wave** [comment]
@merlimat There are ways to write content for Apache Pulsar that would be Vendor Neutral.
1. Perhaps others are contributing to protocol handlers in other ways.

**Comment by sijie** [comment]
@dave2wave Can I get a clarification on the following two links?

[[DeltaLake](https://delta.io/uncategorized/pulsar-in-the-lakehouse-apache-pulsar-with-apache-spark-and-delta-lake/)](https://delta.io/uncategorized/pulsar-in-the-lakehouse-apache-pulsar-with-apache-spark-and-delta-lake/)

[[Druid](https://imply.io/blog/community-spotlight-apache-pulsar-and-apache-druid-get-close/))](https://imply.io/blog/community-spotlight-apache-pulsar-and-apache-druid-get-close/)), [[Trino](https://www.youtube.com/watch?v=Z77DqONXyi4)](https://www.youtube.com/watch?v=Z77DqONXyi4)

These are also links from the other vendors. Are they okay to be included? 

We want to follow the ASF guide.

**Comment by dave2wave** [comment]
@sijie I would rather discuss the approach to Pulsar project blogs so that it becomes easy to follow "The Apache Way"

1. Blog posts should be about one thing. This post should only be about our 500th contributor to the main repository, number of stars, forks, committers, and PMC members. The growth of the community. Other blog posts can be about the technology and focus on adaptors, tiered storage, and the rest.
2. How do we discuss the ecosystem and various vendor efforts and contributions in a fair way. The only way to that is in the open on the dev@pulsar mailing list - we are at the point where no one person can know everything happening in the Pulsar ecosystem. For example, @eolivelli has been working on DataStax's Starligh for Kafka and I've been running OMB benchmarks, we found the major bottleneck and just contributed it back to KoP. If the discussion of Blog posts on dev@pulsar is too much then a marketing@pulsar mailing list could be created.
3. Questions about branding on others Pulsar content can be addressed on a case by case basis by the PMC.

If you would like to follow this path then let's start a discussion on dev@pulsar to plan future blogs and solicit future content from the whole community.

**Comment by sijie** [comment]
@dave2wave thank you for your constructive comment! I agreed that this blog post should just focus on the project growth. @alicebi-streamnative will update the blog post.

**Review by dave2wave — APPROVED** [review]
I will note that Pulsar Summit Global does not yet have a date nor is it fully approved. I would be happy to discuss current plans.

**Review by david-streamlio — APPROVED** [review]

**Pull request merged**